.github/workflows/test-install-plugins.yml

```
@@ -0,0 +1,178 @@
+name: Test Installing Plugins
+
+# Test Top 50 asdf plugins install correctly and the most common can print their
+# current version
+
+on:
+  release:
+    types: [published]
+  schedule:
+    # run at midnight on sunday (utc?)
+    - cron: 0 0 * * 0
+  # TODO: Remove on pull_request once CI is verified to run correctly
+  pull_request:
+    branches: [ "main" ]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-linux:
+    name: Build linux
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: x86_64-unknown-linux-gnu
+          override: true
+      - name: Rust Cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          key: "x86_64-unknown-linux-gnu"
+      - uses: actions-rs/cargo@v1
+        with:
+          use-cross: true
+          command: build
+          args: --release --all-features --target x86_64-unknown-linux-gnu
+      - run: scripts/build-tarball.sh x86_64-unknown-linux-gnu
+      - uses: actions/upload-artifact@v3
+        with:
+          name: tarball-x86_64-unknown-linux-gnu
+          path: |
+            dist/rtx-*.tar.xz
+            dist/rtx-*.tar.gz
+          if-no-files-found: error
+  test-install-and-run:
+    name: Self Test popular plugins
+    runs-on: ubuntu-22.04
+    needs: [ build-linux ]
+    continue-on-error: true
+    env:
+      RTX_MISSING_RUNTIME_BEHAVIOR: autoinstall
+    strategy:
+      matrix:
+        include:
+          - plugin: nodejs
+            command: rtx exec nodejs@latest -- node -v
+          - plugin: ruby
+            command: rtx exec ruby@latest -- ruby --version
+          - plugin: python
+            command: rtx exec python@latest -- python -V
+          - plugin: direnv
+            command: rtx exec direnv@latest -- direnv --version
+          - plugin: erlang
+            command: rtx exec erlang@latest -- erl -eval 'erlang:display(erlang:system_info(otp_release)), halt().'  -noshell
+          - plugin: elixir
+            command: rtx exec elixir@latest erlang -- elixir --version
```

this one isn't working because erlang requires elixir, I think if you swap the order of these it might fix it
```suggestion
            command: rtx exec erlang elixir@latest -- elixir --version
```